web/src/Tooltip.tsx

```diff
@@ -55,22 +64,62 @@ const InfoIcon = styled(InfoSvg)`
     fill: ${Color.gray6};
   }
 `
+const DismissButton = styled(InstrumentedButton)`
+  ${mixinResetButtonStyle};
+
+  .MuiButton-label {
+    font-size: ${FontSize.smallester};
+    font-style: italic;
+    color: ${Color.grayLight};
+  }
+`
```

Comment:
playing around with this locally, i kinda like the dismiss button right-aligned, since it better separates it from the tip content. i'd be curious to hear what @hyu or @SurbhiSGupta think if they have time to look, but i'd be sold either way!

![Screen Shot 2021-08-31 at 2 11 42 PM](https://user-images.githubusercontent.com/23283986/131554803-e9dd6d8b-f96a-4105-8c0a-ca1b6dcf49c8.png)

Reply:
Yes! Do you have the styling for that handy? That's what I initially went for and spent 15 minutes trying to get that to work and decided it wasn't worth the effort at the moment.

Reply:
oh that wasn't so bad!